### PR TITLE
desktop portfolio version 

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project i built a desktop portfolio version webpage with a min width of 768px. grid and flex position elements are used to position elements in my webpage. Also on buttons hover was applied to them .no complex code is done here.
this page is built with :
- HTML
- CSS
- Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # personal-portfolio
 
-In this project i built a personal portfolio page mobile version. In this project i added icons, used the flex box and offset properties where i used to postion ellements and css grid to display the various cards.Also I created a contact form where users can input different informations about themselfs and submit to formspree
-account which i created.no complex code is done here.
+In this project i built a desktop portfolio version webpage with a min width of 768px. grid and flex position elements are used to position elements in my webpage. Also on buttons hover was applied to them .no complex code is done here.
 ## Built With
```
Comment (mgmediaweb) [inline README.md:5]: Kindly, I recommend putting your name in the Readme where it says "Author1"

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

work4
image
image
image
image

## Highlights 💯 
- Descriptive Pull Request ✔️ 
- Professional Readme File ✔️ 

## Required Changes ♻️

Kindly, I suggest you review your website well based on the original design of Figma, I indicate some observations so that you can improve them

image

- Kindly, in the desktop version some details are also appreciated.
image
image

Kindly, in the works section I suggest you implement the hover event on each project
image

image

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (franclobo) [reply README.md:5]: - [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

**Comment by Ngufor-emmanuella** [comment]
Thank you

File: style.css [hunk 2]
```diff
@@ -1,554 +1,1111 @@
...
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+/* styling the navbar */
+.navbar-container {
+  background-color: #3c3a39;
+  color: white;
+  padding: 7px 24px;
+  height: 48px;
+}
```
Comment (franclobo) [inline style.css:19]: - [ ] Please 🙏🏽 make sure that the navbar completes all the width, for a better user experience.

- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

image

File: style.css [hunk 3]
```diff
@@ -1,554 +1,1111 @@
...
-
-.ttwork {
-  padding: 0%;
-  margin: 0%;
+  width: 100%;
+  line-height: 2rem;
+  height: 83px;
+  text-align: center;
+  margin-bottom: 10px;
+  color: #1c1a19;
```
Comment (franclobo) [inline style.css:141]: - [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
image

File: style.css [hunk 4]
```diff
@@ -1,554 +1,1111 @@
...
-  text-align: center;
+/* styling the project or work section */
+
+.work-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 386px;
+  padding: 10% 24px 35% 24px;
+  height: auto;
+  gap: 2%;
```
Comment (franclobo) [inline style.css:211]: - [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

image

File: style.css [hunk 5]
```diff
@@ -1,554 +1,1111 @@
...
-  list-style-type: none;
+/* styling the about section */
+
+.about-container {
   display: flex;
-  flex-direction: row;
-  padding: 1rem 0;
+  flex-direction: column;
+  height: auto;
+  margin: 64% 0 0 0;
```
Comment (franclobo) [inline style.css:292]: - [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
image
image

File: style.css [hunk 6]
```diff
@@ -1,554 +1,1111 @@
...
-  padding: 10rem 2rem 6rem 2rem;
-  background: var(--bkcolor);
-  color: var(--txtcolor);
-  background-image: url(Resources/Images/Frame.png);
-  background-repeat: no-repeat;
-  background-position: right 20rem;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  gap: 1.5rem;
```
Comment (franclobo) [inline style.css:397]: - [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
image

File: README.md (hunk 1 above)
Comment (Ngufor-emmanuella) [reply README.md:5]: hi @franclobo thank you for your correction. please do you mean i should add my name to the READme where it says AUTHOR1?

File: style.css [hunk 7]
```diff
@@ -1,554 +1,1111 @@
...
+body section.about-container div > button:focus,
+footer div.icons form > button:focus {
+  background-color: #e05304;
+  color: #f4f1f0;
 }
 
-/* all the display none for mobile */
-nav > :nth-child(3) {
-  display: none;
+@media (min-width: 320px) {
```
Comment (franclobo) [inline style.css:481]: - [ ] **[OPTIONAL]** Good job doing various @media queries in your styles, but we could separate them into two groups of devices 768px or wider. More information about it, [here](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Professional README file ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## - [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (franclobo) [reply README.md:5]: Hello, @Ngufor-emmanuella yes please, every suggestion from the last reviewer is required unless it has the prefix **[OPTIONAL]**. I have already finished my review, you can check it out now. 😃

**Pull request merged**